doc/api/wasi.md

```diff
@@ -23,12 +23,10 @@ const wasi = new WASI({
 });
 const importObject = { wasi_snapshot_preview1: wasi.wasiImport };
 
-(async () => {
-  const wasm = await WebAssembly.compile(fs.readFileSync('./demo.wasm'));
-  const instance = await WebAssembly.instantiate(wasm, importObject);
+const wasm = await WebAssembly.compile(fs.readFileSync('./demo.wasm'));
```

Comment:
since top level async await not available to all the versions and hehind a flag for now. Maybe not the right time.

Reply:
It isn't flagged anymore, seems fine to use.